gix-status/tests/fixtures/generated-archives/.gitignore

```diff
@@ -1,3 +1,4 @@
 status_unchanged.tar
 status_changed.tar
 symlink_stack.tar
+status_nonfile.tar
```

Comment:
Interestingly, I ran into this locally and on MacOS it definitely didn't properly retain the `pipe` status of the non-files, or it couldn't recreate it. They ended up as normal files after extracting them from the `tar` archive.

Since the script doesn't do much, I think it's also fine to not have it cached.

Reply:
If a commit was pushed, that might still be accessible, that has the `.tar` file, then I could look at it to see if the FIFO didn't end up in it, or if the issue was extraction. (But if not, or if it's not convenient to find, then I could still possibly reproduce this, next time I have access to a macOS system.)